ui/src/CreateMeme.tsx

```diff
@@ -8,10 +8,20 @@ import "./CreateMeme.css";
 import MemeUpload from "./MemeUpload";
 import ImageUpload from "./ImageUpload";
 import Button from "react-bootstrap/Button";
+import { useCookies } from "react-cookie";
+import { useHistory } from "react-router-dom";
 
 function CreateMeme() {
   const [appState, dispatch] = useReducer(AppDispatchReducer, initialState);
   const value = { state: appState, dispatch };
+  // eslint-disable-next-line
```

Comment:
Typescript was flagging `cookies` in line 18 as not being used, but it is required so that `setCookie` may be used. This comment prevents `cookies` from being flagged as a typescript error